src/components/TopBar/AboutDialog.js

```diff
@@ -0,0 +1,359 @@
+                                                                                'about-dialog/git-version'
+                                                                            }
+                                                                        />
+                                                                    }
+                                                                    arrow
+                                                                >
+                                                                    <Chip
+                                                                        icon={
+                                                                            <Bookmark />
+                                                                        }
+                                                                        variant="filled"
+                                                                        size="small"
+                                                                        label={
+                                                                            cmpnt.gitTag
+                                                                        }
+                                                                    />
+                                                                </Tooltip>
+                                                            )}
+                                                            {cmpnt.license && (
+                                                                <Tooltip
+                                                                    title={
+                                                                        <FormattedMessage
+                                                                            id={
+                                                                                'about-dialog/license'
+                                                                            }
+                                                                        />
+                                                                    }
+                                                                    arrow
+                                                                >
+                                                                    <Chip
+                                                                        icon={
+                                                                            <Gavel />
+                                                                        }
+                                                                        variant="outlined"
+                                                                        size="small"
+                                                                        label={
+                                                                            'License: ' +
```

Comment:
For me either add "Tag" for the tag and remove the tooltip message which become useless or just remove "License" here, because it's not really coherent.